### PR TITLE
Fix Missing help article

### PR DESCRIPTION
**Pull request by dogyun-slayte**
- Mail introduction article missing
 - Admi layout concise style

### CODE REVIEW

File: assets/inapp.json [hunk 1]
```diff
@@ -290,5 +290,13 @@
   "Component_DateRangePicker": {
     "name": "Date Range Picker",
     "description": "Pick two dates, begin- and end-date, by navigating back and forth and clicking on the dates of choice. You can also directly enter the dates using your keyboard - Simply type the date in the format 'MMM D, YYY', e.g. Sep 1, 1985. See below:\n[img height=451 width=564]http://support-static.slayte.com/component-daterangepicker-format.gif[/img]"
+  },
+  "Mail_Introduction": {
+    "name": "Mail Introduction",
+    "description": "Message Templates can be used to create default messages that will be sent to users depending on an event type of your choice."
```
Comment (maximilianvp) [inline assets/inapp.json]:
```
Use Message Templates to build custom and re-usable messages with mail-merge placeholders. Each Message Template is built for a specific type of event, e.g. "Submission Received", which sets the available placeholders.
```

File: assets/inapp.json [hunk 2]
```diff
@@ -290,5 +290,13 @@
...
+  },
+  "Admin_ConciseStyle": {
+    "name": "Admin Concise Style",
+    "description": " This option creates a more concise-looking Submission Form and reduces the line spacing on the Submission Form page."
```
Comment (maximilianvp) [inline assets/inapp.json]:
```
Using concise style will compress the layout of the Submission Form and reduce overall spacing.
```

File: assets/inapp.json (hunk 1 above)
Comment (dogyun-slayte) [reply assets/inapp.json]: changed

File: assets/inapp.json (hunk 2 above)
Comment (dogyun-slayte) [reply assets/inapp.json]: changed

**Comment by dogyun-slayte** [comment]
Fix conflicts

**Pull request merged**